starknet-foundry/crates/cast/src/starknet_commands/multicall/mod.rs

```diff
@@ -0,0 +1,30 @@
+use camino::Utf8PathBuf;
```

Comment:
What are the upsides of having this file split vs having `new` and `run` in the same file? Is it a common rust practice, to hold only some structs/enums in mod file?

Reply:
Clear structure. We have a similar one in tests, here it works just fine too I think.

Reply:
That's fine. Can you move the specific Command enums to their respective files though? @war-in rightly noticed that that would bring even more structure clarity, and I agree with this. 

To be explicit - I mean having `Run` commands enum defined in run.rs, and `New` in new.rs. What do you think?

Reply:
[move structs](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/627fdfbba6147c655662f4a99c331e74899ec082)